tests/integration/convert_and_import_excel_file.t

```diff
@@ -124,16 +147,17 @@ sub fake_download_image ($) {
 			push(@sto_products, retrieve_product($product->{code}));
 		}
 		normalize_products_for_test_comparison(\@sto_products);
-		compare_array_to_expected_results(\@products, $expected_result_dir, $update_expected_results);
+		compare_array_to_expected_results(\@products, $expected_test_results_dir, $update_expected_results);
 	}
 
-	compare_to_expected_results($stats_ref, $expected_result_dir . "/stats.json", $update_expected_results);
+	compare_to_expected_results($stats_ref, $expected_test_results_dir . "/products/stats.json",
+		$update_expected_results);
 
 	# TODO verify images
 	# clean csv and sto
 	unlink $inputs_dir . "eco-score-template.xlsx.csv";
 	unlink $inputs_dir . "test.columns_fields.sto";
-	rmdir remove_tree($outputs_dir);
+	#rmdir remove_tree($outputs_dir);
```

Comment:
shan't we remove outputs dir ?

Reply:
ah yes, I commented it for debugging and forgot it